### ISSUE TITLE
New SC for audio quality of canned video/audio?

### ISSUE THREAD

**Issue by detlevhfischer** [w3c/wcag3#173]
Hi,
Recently, a user enraged about the abysmal audio quality of some info video on a federal website enquired whether audio quality is part of the accessibility requirements of the German directive (which points to the EN 301 549 and via that to WCAG 2.1 and additional stuff). 
The 'additional stuff' part actually checks the bandwidth aspect of audio quality in EN 301 549  requirement "6.1 Audio bandwidth for speech", but if the audio source quality is bad I guess this is a separate problem.
It stands to reason that bad audio quality would disproportionally affect people with hearing impairments, so I think in principle such a requirement (and SC) might qualify. I am not aware, however, if there are good automated tools to check audio (speech) quality and where you would draw the line between pass and fail. I'd be curious to hear other people's opinion on that.
